setup.cfg

```diff
@@ -35,6 +35,8 @@ install_requires =
     click-didyoumean
     dandischema >= 0.11.0, < 0.12.0
     etelemetry >= 0.2.2
+    # For pydantic to be able to use type annotations like `X | None`
```

Comment:
Is there a reason you can't just use `Union` and `Optional` instead?

Reply:
No, but the code base already had many uses of `|` to express `Optional` and `Union` types. I only made the models in `validate_types.py` to Pydantic models, and since Pydantic uses the types at runtime, the `eval_type_backport` package is needed. To avoid the `eval_type_backport` dependency, some code unrelated to the purpose of this PR will need to change.